client/app/components/preprint-basics/component.js

```diff
@@ -100,6 +130,15 @@ export default Ember.Component.extend(BasicsValidations, {
                 return;
             }
 
+            // Save locally
+            let values = {
+                basicsDOI:  this.get('basicsDOI'),
+                basicLicense: this.get('basicsLicense'),
+                basicsTags: this.get('basicsTags').map(function(a){ return a;}),
```

Comment:
It took me a second to realize this `map` function is just copying the array. Why not use `Ember.$.extend(true, [], this.get('basicsTags'));`?